React/Views/RCTModalHostView.m

```diff
@@ -101,6 +134,11 @@ - (void)removeReactSubview:(UIView *)subview
   RCTAssert(subview == _reactSubview, @"Cannot remove view other than modal view");
   [super removeReactSubview:subview];
```

Comment:
We have to add a comment here about that fact that super method actually removes `subview` from `_modalViewController.view` because it absolutely not obvious.